src/components/login/login.js

```diff
@@ -0,0 +1,57 @@
+const logIn = () => {
+  localStorage.setItem("loggedIn", true);
+  window.location.href = "/allReviews";
+};
+
+// BUILD AND RENDER LOGIN FORM
```

Comment:
Where are these functions going to be called? (ie. How is this being incorporated into the rest of the App?)